packages/mock-block-dock/src/assets/debug-view-styles.module.css

```diff
@@ -27,17 +28,23 @@
   src: url("./jet-brains-mono-regular.woff2") format("woff2");
 }
 
-.mbd-non-debug-mode-wrapper {
-  padding-top: 60px;
+body {
+  margin: 0;
+}
```

Comment:
Is this allowed downstream? E.g. in Next.js

Reply:
No – it seems we will have to inline these styles. I will do it in a follow up since the problem exists already